tests/README.md

```diff
@@ -92,6 +92,17 @@ Some helpful commands:
     $ oc describe pods
     $ oc log -f cockpit-tests-xxxx
 
+The tests need ```/dev/kvm``` to be accessible to non-root users on each node:
+
+    $ sudo chmod 666 /dev/kvm
+
+Some tests need nested virtualization enabled:
+
+    $ sudo -s
+    # echo "options kvm-intel nested=1" > /etc/modprobe.d/kvm-intel.conf
+    # echo "options kvm-amd nested=1" > /etc/modprobe.d/kvm-amd.conf
+    # ( rmmod kvm-intel && modprobe kvm-intel ) || ( rmmod kvm-amd && modprobe kvm-amd )
```

Comment:
Other Openshift clusters need this information. This stuff is for Openshift nodes. 

Longer term @dperpeet is going to come up with a smoother ansible way to make this stuff stick across the Openshift nodes. Cockpit's test/vm-prep is about to be deleted. The container setup script does a bunch more than just this. I'd suggest just waiting until this has a chance to settle before we optimize here.